docs/content/en/docs/design/config.md

```diff
@@ -105,7 +102,19 @@ deploy:
   # deploy definition
 ```
 
-Configurations `cfg1` and `cfg2` from the above file can be imported as dependencies in your current config, via:
+Here `cfg1` and `cfg2` are independent skaffold configurations. Running `skaffold dev` for instance will execute actions from both these configurations. You could also run `skaffold dev -m cfg1` to only activate the `cfg1` configuration and skip `cfg2`.
+
+## Configuration dependencies
+
+In addition to authoring configurations in a `skaffold.yaml` file, we can also import other existing configurations as dependencies. Skaffold manages all imported and defined configurations in the same session. It also ensures all artifacts in a required config are built prior to those in current config (provided the artifacts have dependencies defined); and all deploys in required configs are applied prior to those in current config.
```

Comment:
```suggestion
In addition to authoring configurations in a `skaffold.yaml` file, we can also import other existing modules as dependencies. Skaffold manages all imported and defined modules in the same session. It also ensures all artifacts in a required config are built prior to those in current config (provided the artifacts have dependencies defined); and all deploys in required modules are applied prior to those in current config.
```

Reply:
I've changed all other occurrences of `modules` vs `configurations`. The distinction is only important for the `--module` flag which requires named-configurations. So a module is then a named configuration.